docs/directives.mjs

```diff
@@ -0,0 +1,154 @@
+import { u } from 'unist-builder';
+import { mystParse } from 'myst-parser';
+import { defaultDirectives } from 'myst-directives';
+import { fileError } from 'myst-common';
+
+export const plugin = {
+  name: 'MyST Documentation Plugins',
+  author: 'Rowan Cockett',
+  license: 'MIT',
+};
+
+/**
+ * @param {import('myst-common').OptionDefinition} option
+ */
+function type2string(option) {
+  if (option.type === 'string' || option.type === String) return 'string';
+  if (option.type === 'number' || option.type === Number) return 'number';
+  if (option.type === 'boolean' || option.type === Boolean) return 'boolean';
```

Comment:
@fwkoch I extended this based on markdocs API which I quite liked the simplicity of. You can now say an option is of type string with 'String' instead of `ParsedENumType.string`. Very small change, but looks a bit better, backwards compatible as well.